site/src/pages/tutorials/grpc/blog/implement-read.mdx

```diff
@@ -63,8 +67,8 @@ import java.util.Collections;
 import java.util.Comparator;
 import java.util.Map.Entry;
 
-import example.armeria.blog.grpc.Blog.ListBlogPostsRequest;
-import example.armeria.blog.grpc.Blog.ListBlogPostsResponse;
+import example.armeria.blog.grpc.ListBlogPostsRequest;
```

Comment:
👍 👍 👍 I like how we're finding these bugs while restructuring the tutorial.

I think it also helps that the overall tutorial will be more in-sync with the code